synapse/lib/view.py

```diff
@@ -1387,6 +1387,11 @@ async def addTrigger(self, tdef):
 
         root = await self.core.auth.getUserByName('root')
 
+        # preserve the created timestamp in case of move
+        created = tdef.get('created')
+        if created is None:
+            tdef['created'] = s_common.now()
+
```

Comment:
This could also be implemented as `tdef.setdefault('created', s_common.now())` to keep things terse